Headers/Public/VLCMediaPlayer.h

```diff
@@ -147,7 +147,13 @@ extern NSString * VLCMediaPlayerStateToString(VLCMediaPlayerState state);
  * @param player the player who stopped recording
  * @param path the path to the file that the player recorded to
  */
-- (void)mediaPlayer:(VLCMediaPlayer *)player recordStoppedAtPath:(NSString *)path;
+- (void)mediaPlayer:(VLCMediaPlayer *)player recordingStoppedAtPath:(NSString *)path;
+
+/**
+ * Sent by the default notification center whenever the player recording changed.
+ * @param arguments first ojects contains the "filePath" and the "isRecording" field
+ */
+- (void)mediaPlayerRecordChanged:(NSArray *)arguments;
```

Comment:
I just noticed the arguments here and have to say that I'd rather have the bool and filepath separatly.
@Mikanbu  Is there any reason it's arguments in the first place ?

Reply:
IIRC, it's based on the `VLCEventManager` class and this method shouldn't be exposed.

Reply:
also why is this an array that contains a dict and not immediatly a dict?

Reply:
@Mikanbu ah I found the correct place then